spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/DefaultAmqpHeaderMapper.java

```diff
@@ -59,7 +59,7 @@
  */
 public class DefaultAmqpHeaderMapper extends AbstractHeaderMapper<MessageProperties> implements AmqpHeaderMapper {
 
-	static final boolean CONSUMER_METADATA_PRESENT;
+	private static final boolean CONSUMER_METADATA_PRESENT;
```

Comment:
I think we can remove this altogether and make S-A 2.0 the minimum for S-I 5.0.

WDYT?